### DISCUSSION TITLE
Valore degli stati di una fattura elettronica (ei_status)

### DISCUSSION THREAD

**Discussion by Sciabal** [fattureincloud/api#262] [Q&A] [answered]
Ciao,

forse sono io che non so dove cercare, ma non riesco a trovare i possibili valori stringa che può assumere il campo **ei_status**, in risposta alla chiamata issued_documents.
Sino ad ora ho riscontrato i seguenti:

- Non inviata -->  valore stringa "**not_send**"
- Inviata --> valore stringa "**send**"
- In gestione --> valore stringa "**processing**"
- In attesa --> valore stringa "**pending**"
- Controlli in corso --> valore stringa ???
- _Altri?_

Dove trovo la lista completa?
Inoltre, qual'è la differenza sostanziale tra "processing" e "pending" (al momento ho 2 fatture segnate come "In gestione" su FiC ma se le chiamo con le API una risponde "pending" e l'altra "processing" ... )

Grazia
Ciao!

**Comment by emajo** [comment, marked as answer]
Ciao @Sciabal, questi stati si possono trovare sulla nostra [api reference](https://developers.fattureincloud.it/api-reference#post-/c/-company_id-/issued_documents), allego anche l'immagine relativa.


![status](https://user-images.githubusercontent.com/48432054/216580921-e1569d09-e9cb-4f1f-a1ae-44e34841a3eb.png)

Grazie,
Emanuele.

**Reply by Sciabal** [reply]
Ah caspita, che sciocco! Mi ero scordato del tab schema...
Grazie e scusate la banalità!

Ciao!
